### DISCUSSION TITLE
Error when using -format in Neper to convert mesh into .inp file for ABAQUS

### DISCUSSION THREAD

**Discussion by amgamo1** [neperfepx/neper#305] [Q&A] [answered]
Hello,

I'm trying to get used to Neper and how to use it with ABAQUS. Following the examples in the manual, i generated a tesselation and mesh it with

$ neper -T -n 10 -id 1
$ neper -M n10-id1.tess -format inp

When importing it to ABAQUS 2021 (student ver), i get this error below: 
"AbaqusException: in keyword *ELSET, file "n10-id1.inp", line 2255: The following sets specified do not exist. Sets should be defined before they are used:
TESS.113"

And the error's repeated for every 2 lines after that until the next Elset syntaxes(i.e line 2257 has an error, 2259 has an error ,etc until 2491. Then it jumps to the next Elsets at line 2613 with error ) . 

So how do i properly import an inp output file from Neper into ABAQUS? 

Unfortunately, i cant attach an inp file but here's the link to the inp file : https://drive.google.com/file/d/1UkOGihsydvItXqoO8Vyz86xaQQ6MgEN7/view?usp=sharing

Thank you in advance for any help

**Comment by rquey** [comment, marked as answer]
I've just fixed this in version `4.2.1-11` (see the `main` branch on the GitHub repository). Otherwise, use `-faset none`.
